framework/src/test/java/org/checkerframework/framework/testchecker/sideeffectsonly/qual/Refined.java

```diff
@@ -0,0 +1,19 @@
+package org.checkerframework.framework.testchecker.sideeffectsonly.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * Bottom qualifier of a toy type system. The toy type system is used to test whether dataflow
```

Comment:
Rather than "Refined", I suggest giving this a more specific name, such as "SideEffectsOnlyToyBottom" (or some better name of your choosing).  Likewise for the top qualifier.